### PR TITLE
added optional nodeport variable

### PR DESCRIPTION
**Pull request by ethanwinograd**
will default to ClusterIP if not provided.

### CODE REVIEW

File: util/k8sutil/k8sutil.go [hunk 1]
```diff
@@ -426,6 +426,11 @@ func (k *K8sutil) CreateClientService() error {
 				},
 			},
 		}
+		if nodePort > 0 {
+			clientSvc.Spec.ClusterIP = v1.ClusterIPNone
```
Comment (stevesloka) [inline util/k8sutil/k8sutil.go]: Need to delete this line. There's a conflict when you set type to `NodePort` then set clusterIP to none.

**Comment by ethanwinograd** [comment]
committed with that line removed

**Comment by stevesloka** [comment]
Looks like my commits causes now a merge conflict, do you mind rebasing and pushing resovling the issue? Thanks!

**Comment by ethanwinograd** [comment]
I created a new PR for this under a new branch

**Pull request closed without merging**